language-support/ts/daml-ledger/index.ts

```diff
@@ -502,7 +502,7 @@ class Ledger {
     const protocols = ['jwt.token.' + this.token, 'daml.ws.auth'];
     let ws = new WebSocket(this.wsBaseUrl + endpoint, protocols);
     let isLiveSince: undefined | number = undefined;
-    let lastOffset: undefined | string = undefined;
+    let lastOffset: undefined | null | string = undefined;
```

Comment:
Do we have any case for distinguishing between undefined and null here? If not, could we simplify the type and not allow undefined, and start with `null` as the initial value?

Reply:
Quoting the comment from the source:
```
      // we try to reconnect if we could connect previously and we were live for at least
      // 'reconnectThreshold'.
```
So there is a difference, we will reconnect with `null` but not with `undefined`. The distinction here seems meaningful so I would argue for keeping it.